pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/JWTRouteFilter.java

```diff
@@ -38,27 +41,23 @@
      * secret, and optionally includes one or more of the
      * given roles in the roles claim.
      *
-     * @param jwtSecret    the shared secret used to sign the token
      * @param allowedRoles If given, at least one of the roles in the list
      *                     must be present in the roles claim in the token.
      *                     If empty or null, the roles claim is not checked.
      */
-    public JWTRouteFilter(String jwtSecret, Collection<String> allowedRoles) {
-        if (StringUtils.isBlank(jwtSecret)) {
-            throw new IllegalArgumentException("jwtSecret is required");
-        }
-        this.jwtSecret = jwtSecret;
+    public JWTRouteFilter(Collection<String> allowedRoles, String auth0Domain, String bspSecret) {
```

Comment:
JWTRouteFilter is always called with allowedRoles = null, so you could just remove that from the class

Reply:
all removed